### DISCUSSION TITLE
correlateMatrices with integrated snRNA Error: no mapping

### DISCUSSION THREAD

**Discussion by jacobHog** [GreenleafLab/ArchR#1323] [Questions / Documentation]
Hello,
I am trying to identify positive TF-regulators of my constrained integrated RNA using ArchR. 
The constrained integration is completed using:

Proj1 <- addGeneIntegrationMatrix( 
    ArchRProj = Clean_merged_3,  
    useMatrix = "GeneScoreMatrix", 
    matrixName = "GeneIntegrationMatrix", 
    reducedDims = "IterativeLSI", 
    seRNA = seRNA, 
    addToArrow = TRUE,  
    groupList = groupList, 
    groupRNA = "celltype", 
    sampleCellsATAC = 2500, 
    sampleCellsRNA = 2500, 
    nameCell = "predictedCell_Co", 
    nameGroup = "predictedGroup_Co", 
    nameScore = "predictedScore_Co", 
    genesUse=c, 
    force=TRUE 
) 

Where "Clean_merged_3" is my ATAC-seq data cleaned and "seRNA" is my RNA-seq data cleaned.
```
> **Clean_merged_3**
           ___      .______        ______  __    __  .______
          /   \     |   _  \      /      ||  |  |  | |   _  \
         /  ^  \    |  |_)  |    |  ,----'|  |__|  | |  |_)  |
        /  /_\  \   |      /     |  |     |   __   | |      /
       /  _____  \  |  |\  \\___ |  `----.|  |  |  | |  |\  \\___.
      /__/     \__\ | _| `._____| \______||__|  |__| | _| `._____|

class: ArchRProject
outputDirectory: /data/Grontved/students/JBH/snATAC_ArchR_results/All_Merged/Wat
samples(8): 6580 6581 ... 6590 6591
sampleColData names(1): ArrowFiles
cellColData names(18): Sample TSSEnrichment ... Clustering Clus
numberOfCells(1): 54899
medianTSS(1): 20.008
medianFrags(1): 4406


> **seRNA**
An object of class Seurat
21980 features across 95879 samples within 1 assay
Active assay: RNA (21980 features, 2000 variable features)
 4 dimensional reductions calculated: pca, umap, harmony, umap_reference
```

My issue is that when I am performing Identification of Positive TF-regulators (15.4 Full Manual) I get the following error:

_Code:_
```
seGroupMotif <- getGroupSE(ArchRProj = Proj1, useMatrix = "GeneIntegrationMatrix", groupBy = "Clus") 

seZ <- seGroupMotif[rowData(seGroupMotif)$seqnames=="z",] 

rowData(seZ)$maxDelta <- lapply(seq_len(ncol(seZ)), function(x){ 
  rowMaxs(assay(seZ) - assay(seZ)[,x]) 
}) %>% Reduce("cbind", .) %>% rowMaxs 

corGIM_MM <- correlateMatrices( 
    ArchRProj = Proj1, 
    useMatrix1 = "GeneIntegrationMatrix", 
    useMatrix2 = "MotifMatrix", 
    reducedDims = "IterativeLSI" 
) 
```
_Code end_

Output: 
ArchR logging to : ArchRLogs/ArchR-correlateMatrices-3f182cbede25-Date-2022-03-08_Time-11-56-49.log
If there is an issue, please report to github with logFile!
Error: Matching of useMatrix1 and useMatrix2 resulted in no mappings!

Why is this error shown? Do I have to apply some additional functions on the Proj1 object before starting with the positive TF-regulators analysis? 

Thanks

**Comment by rcorces** [comment]
please post errors to Issues. I will create an issue from this discussion
